### DISCUSSION TITLE
How to call VendorDirectFulfillmentOrders in your SDK

### DISCUSSION THREAD

**Discussion by syedjamil12** [abuzuhri/Amazon-SP-API-CSharp#218] [Q&A]
Abu Zuhri,

I tried to Call VendorDirectFulfillmentOrders in your C# SDK but I failed to create call can you give code example for VendorDirectFulfillmentOrders asap.

**Comment by abuzuhri** [comment]
can you descript what you mean with failed do you get error ?
